### PR TITLE
US-512993 : Adding the capability to configure servicetype.

### PR DESCRIPTION
**Pull request by anuds20**
Added the capability to specify the service type. Also added a capability to restrict access to specific IP CIDR blocks in case of a LoadBalancer service type.

### CODE REVIEW

File: charts/pega/values.yaml [hunk 1]
```diff
@@ -162,12 +162,16 @@ global:
...
             insecureSkipVerify: false
-        serviceType: "LoadBalancer"
-        # Specify the CIDR range inside the qoutations to restrict the service access to the given CIDR range
-        # e.g "123.123.123.0/24"
-        loadBalancerSourceRanges:
-          - "103.38.88.0/24"
-          - "49.43.219.90/32"
+        # Use this parameter to deploy service of a specifictype. This is an optional value and should be used based on the usecase.
+        # serviceType: ""
+        # Should be used only when serviceType is set to LoadBalancer.
```
Comment (taz-pega-work) [inline charts/pega/values.yaml:167]: use spacing between words. If servicetype requires, LoadBalancer without a space, you have to call that out.

Is this the context? I'm guessing here...please add any missing details>
To specify a load balancing service, uncomment the previous line to enable its use and specify, `serviceType:` "LoadBalancer".

File: charts/pega/values.yaml [hunk 2]
```diff
@@ -162,12 +162,16 @@ global:
...
+        # Specify the CIDR ranges to restrict the service access to the given CIDR range.
+        # Each new CIDR block should be added in a separate line.
```
Comment (taz-pega-work) [inline charts/pega/values.yaml:169]: Uncomment the following lines and add each CIDR block in a separate line using double quotes, replacing the example with your configuration requirements.

**Review by taz-pega-work — CHANGES_REQUESTED** [review]
Please update the inline comments to be a little more precise and helpful to clients. I hope this helps.

File: charts/pega/values.yaml (hunk 1 above)
Comment (anuds20) [reply charts/pega/values.yaml]: Made changes according to the comments.

**Review by taz-pega-work — APPROVED** [review]
looks much better. One more small change would be helpful if you can make the change on line 144>>
For example if you want to deploy a service of type LoadBalancer, uncomment the following line and specify serviceType: "LoadBalancer"
        # serviceType: ""

File: charts/pega/values.yaml (hunk 2 above)
Comment (anuds20) [reply charts/pega/values.yaml]: Made changes according to the comment.

**Comment by anuds20** [comment]
> looks much better. One more small change would be helpful if you can make the change on line 144>> For example if you want to deploy a service of type LoadBalancer, uncomment the following line and specify serviceType: "LoadBalancer" # serviceType: ""

Made changes according to the comment

**Review by MadhuriArugula — APPROVED** [review]

**Pull request merged**